raw-migrated-files/observability-docs/observability/apm-getting-started-apm-server.md

```diff
@@ -63,9 +63,9 @@ Install, configure, and run the APM Server binary wherever you need it.
 
 **Supported outputs**:
 
-* {es}
+* {{es}}
 * {ess}
```

Comment:
```suggestion
* {{ess}}
```

Reply:
ugh swore I fixed that one!!!!

Reply:
Pretty sure we'll keep adding some back bc of old habits 😅